generated/nidcpower/nidcpower/_grpc_stub_interpreter.py

```diff
@@ -0,0 +1,502 @@
+# -*- coding: utf-8 -*-
+# This file was generated
+
+import grpc
+import hightime  # noqa: F401
+import threading
+import warnings
+
+from . import enums as enums
```

Comment:
Based on the TravisCI errors, it looks like we're going to need to add the `# noqa: F401` comment here (specifically, in the .mako).

Reply:
Done!